tools/seqtk_nml/seqtk_nml.xml

```diff
@@ -1,113 +1,114 @@
-<tool id="seqtk_nml_sample" name="seqTK Sample NML" version="1.0.0">
+<tool id="seqtk_nml_sample" name="seqTK Sample NML" version="1.0.1">
   <description>Runs seqTK sample if raw coverage is above user defined threshold </description>
   <requirements>
     <requirement type="package" version="1.2">seqtk</requirement>
     <requirement type="package" version="1.6.924">perl-bioperl</requirement>
   </requirements>
-  <stdio>
-    <exit_code range="1:" level="fatal" description="Unknown error has occured"/>
-  </stdio>
-  <command>
-		perl  $__tool_directory__/seqtk_nml.pl --ref $fastar 
+  <command detect_errors="exit_code"><![CDATA[
+    perl  $__tool_directory__/seqtk_nml.pl 
```

Comment:
single quotes around $__tool_directory__/seqtk_nml.pl

Reply:
Ah, just when I thought I got all the single quotes. Thanks for catching that!